docs/gds-supported-platform/per-namespace-gateway-opt-in.md

```diff
@@ -0,0 +1,49 @@
+# Per-namespace istio gateways
+
+We are migrating from using a single `ingressgateway` in `istio-system` to an `ingressgateway` in each namespace. You must opt into / enable an ingressgateway in each namespace via `*-cluster-config` values.
```

Comment:
> We are migrating

This documentation will not age well.  Is it possible to write the document with the new world in mind, and maybe have a little migration guide section?

Secondly, *why* might a user wish to opt in to this? What's in it for them? Is it worth a sentence or two of motivation?

Reply:
@philandstuff Will there be a period of time when a user could do either the old world or the new world? Or will there will a hard stop to the old world and a hard start to the new world? 

Regardless, best practice is to document both. If the first, then say "If you are doing x, do y. If you are doing a, do b." If the second, have the old world in the doc, and then switch over when the new world starts.

Reply:
See my other comments / the latest commit. I restructured this document a bit to change how it approaches the info.